### DISCUSSION TITLE
How can I get the markdown output out of an editor?

### DISCUSSION THREAD

**Discussion by juliogarciag** [facebook/lexical#1770] [Q&A] [answered]
I'm using the `MarkdownShortcutPlugin` so when I type `# Title` it gets styled like a title. I couldn't find a way to get the markdown output of the editor. I noticed this text in the README of the [lexical-markdown](https://github.com/facebook/lexical/tree/main/packages/lexical-markdown) package:

> It exports Lexical to a plain text with markup. See convertToPlainTextUtils.js

But that file doesn't exist, so I guess it's still not there yet, right?

**Comment by fantactuka** [comment]
Markdown import/export work is in progress, I'd expect it to be more less in shape within next couple of weeks

**Reply by qianzez** [reply]
Hi! I was wondering if there was an update on this?

**Reply by qianzez** [reply]
Nevermind, i see that the import/export for markdown is there. Thanks.

**Reply by sugianto21** [reply]
how can i change the markdown converter so its can similar with whatsapp markdown?

**Reply by fantactuka** [reply]
You can provide your own list of custom markdown transformers to import/export/shortcuts, see docs link above

**Reply by sugianto21** [reply]
hi @fantactuka i have transform the lexical state to markdown but the `lineBreakNode` transform to `\n\n` if i enter twice it become `\n\n\n\n` i just want it to be `\n` if i enter once and `\n\n` if i enter twice or so on. how to fix this. and how to custom the `lineBreakNode` transform?

**Comment by fantactuka** [comment, marked as answer]
Markdown import/export and shortcuts docs: https://lexical.dev/docs/packages/lexical-markdown
